src/xo-app/home/index.js

```diff
@@ -337,7 +337,7 @@ export default class Home extends Component {
         <Col mediumSize={6}>
           <div className='input-group'>
             <div className='input-group-btn'>
-              <DropdownButton id='filter' bsStyle='secondary' title={_('homeFilters')}>
+              <DropdownButton id='filter' bsStyle='info' title={_('homeFilters')}>
```

Comment:
`secondary` (which disappeared in Bootstrap 4), `danger`, `info` are semantic classes, they should not be used to give a specific color but to express semantic.